easybuild/easyconfigs/g/GPAW/GPAW-1.2.0-foss-2016b-Python-3.5.2.eb

```diff
@@ -0,0 +1,31 @@
+easyblock = "PythonPackage"
+
+name = 'GPAW'
+version = '1.2.0'
+
+homepage = 'https://wiki.fysik.dtu.dk/gpaw/'
+description = """GPAW is a density-functional theory (DFT) Python code based on the projector-augmented wave (PAW)
+method and the atomic simulation environment (ASE). It uses real-space uniform grids and multigrid methods or 
+atom-centered basis-functions."""
+toolchain = {'name': 'foss', 'version': '2016b'}
+source_urls = [PYPI_LOWER_SOURCE]
+sources = [SOURCELOWER_TAR_GZ]
+
+patches = ['GPAW-1.2.0-customize.patch']
+python = 'Python'
+pythonver = '3.5.2'
+versionsuffix = '-%s-%s' % (python, pythonver)
```

Comment:
please use the `%(pyver)s` template like we do in other easyconfigs, and move up the `versionsuffix` definition right under `version`